README.md

```diff
@@ -0,0 +1,80 @@
+# Cucumber.ml
+
+This implements the core Cucumber feature file language, Gherkin, and
+assocated library for specifying the execution of those tests for the
+OCaml programming language.
+
+WARNING: This is still under heavy development and is provided as-is.
+This is for the adventurous who do not mind rough edges and
+half-implemented features.  All pull-requests are gratefully accepted.
+
+## libgherkin.so
+
+To be able to run the code, you will need to have compiled and
+installed the Gherkin library as a shared object in your Linux
+distribution (so that it will be avaliable to the linker at run time).
+You can do this by checking out the Cucumber project from Github
+[Cucumber](https://github.com/cucumber/cucumber) and applying pull
```

Comment:
We prefer directing consumers to the subrepo: https://github.com/cucumber/gherkin-c

Also - see my comment in https://github.com/cucumber/cucumber/pull/321 about creating a new PR in this repo (to iron out a few build kinks)